open_spiel/observer.cc

```diff
@@ -118,15 +118,26 @@ std::shared_ptr<Observer> Game::MakeObserver(
 
   // Perfect information games provide public information regardless
```

Comment:
This comment is no longer accurate.

Reply:
Why?

Reply:
Because NoPrivateObserver does not provide public information.